cmd/kubeadm/app/util/etcd.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"github.com/coreos/etcd/clientv3"
+	"time"
+)
+
+// GetEtcdClusterStatus returns nil for status Up or error for status Down
+func GetEtcdClusterStatus() (*clientv3.StatusResponse, error) {
+	ep := []string{"localhost:2379"}
```

Comment:
nit: get this parameter as an argument for reusability maybe?

Reply:
if we start passing here endpoints parameter, it would complicate this func right away as some check and validations needs to be added. Right now it is bulletproof. I would suggest to leave it as is and if reusability case comes up, refactor it then.

Reply:
ok